### PR TITLE
DOC-1172 Versioning scheme page

### PR DESCRIPTION
**Pull request by ernopp**
- Adds new versioning scheme page
- Approved by DavidP & reviewed by Laura [in this Gdoc](https://docs.google.com/document/d/1ihSDw23y74vS2S2JUm5v-3pADNAnzfcmqFrbQ_vHl9M/edit#heading=h.o1n4v7p7au8y) - feedback incorporated and PRed directly. 
- Tested in improbadoc

cc @anne-edwards this is a bronze publish

### CODE REVIEW

File: SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+*   ``4.22-SpatialOSUnrealGDK-release`` (default)
+*   ``4.22-SpatialOSUnrealGDK-preview``
+
+when Unreal Engine 4.22 is the version supported.
+
+## Unreal Engine version support
+
+**The GDK has full support for one version of Unreal Engine at a time** - the version that is shown in the Unreal Engine Fork repository’s default branch name, and is indicated in the Get Started documentation.
+This is the only version that the Example Project supports. With version 0.6.0, this is Unreal Engine version 4.22.
```
Comment (oblm) [inline SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md]:
```suggestion
```

Comment (oblm) [reply SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md]: Delete this as my above edit adds Example Project to that sentence. The second part of this line will go out of date very quickly so should not be included here.

File: SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+*   ``4.22-SpatialOSUnrealGDK-preview``
+
+when Unreal Engine 4.22 is the version supported.
+
+## Unreal Engine version support
+
+**The GDK has full support for one version of Unreal Engine at a time** - the version that is shown in the Unreal Engine Fork repository’s default branch name, and is indicated in the Get Started documentation.
+This is the only version that the Example Project supports. With version 0.6.0, this is Unreal Engine version 4.22. 
+
+To enable you to transition, the GDK has legacy support during one release cycle for the Unreal Engine version previously supported. For example, with version 0.6.0, the legacy support is for Unreal Engine 4.20, which is available through the ``4.20-SpatialGDK-release`` branch of the SpatialOS Engine Fork. The Example Project is no longer supported with this version.
```
Comment (oblm) [inline SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md]:
```suggestion
To facilitate a smooth upgrade for your project, when we introduce support for a new Unreal Engine version, we also support the previously Engine version in parallel, for one GDK release. For example, GDK version `0.6.0` introduces `Unreal Engine 4.22` support, but is also the last GDK version to support `4.20-SpatialGDK-release`.

File: SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+We will usually start working on supporting the most recent Unreal Engine minor version after a few patch releases that make that minor version more stable. For instance, we will start working on Unreal Engine 4.23 support for our next GDK release once 4.23.2 or 4.23.3 is available. To see when a new version will be supported, check out the [Development Roadmap](https://github.com/spatialos/unrealgdk/projects/1).
```
Comment (oblm) [inline SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md]: this is non-comittal and not helpful, we should either be declarative (e.g: We endeavour to support the latest stable and recommended version of the Unreal Engine less than 3 months after it's released) or not include this at all.

File: SpatialGDK/Documentation/content/upgrading.md [hunk 4]
```diff
@@ -20,22 +20,24 @@ If you followed our [Get started]({{urlRoot}}/content/get-started/introduction)
 You can find out which branch you have checked out by following the instructions below:<br>
 
 1. In a terminal of your choice, change directory to the root of the repository.<br>
-1. Run `git status`.
+2. Run `git status`.
 This should return `On *-SpatialOSUnrealGDK-release` in your `UnrealEngine` repository and `On release` in your `UnrealGDK` repository.<br>
```
Comment (oblm) [inline SpatialGDK/Documentation/content/upgrading.md]:
```suggestion
This should return `On branch *-SpatialOSUnrealGDK-release` in your `UnrealEngine` repository and `On branch release` in your `UnrealGDK` repository.<br>
```

File: SpatialGDK/Documentation/content/upgrading.md [hunk 5]
```diff
@@ -20,22 +20,24 @@ If you followed our [Get started]({{urlRoot}}/content/get-started/introduction)
...
 
 To update your Unreal Engine fork and GDK to the latest version, complete the following steps:
 
 1. In a terminal, change directory to the root of `UnrealEngine`.
-1. Run `git pull` to update your Unreal Engine.
-1. In a terminal, change directory to the root of `UnrealGDK`.
-1. Run `git pull` to update your GDK.
-1. Open **File Explorer**, navigate to the root directory of the Unreal GDK repository, and then double-click **`Setup.bat`**. You might be prompted to sign into your SpatialOS account if you have not signed in yet.
-1. In **File Explorer**, navigate to the `<GameRoot>` directory that contains your project's `.uproject` file.<br>
+2. Run `git pull` to update your Unreal Engine.
```
Comment (oblm) [inline SpatialGDK/Documentation/content/upgrading.md]: Has your IDE erroneously auto corrected these?

We use 1.1.1. lists apart from in very limited circumstances where they don't render correctly?

File: SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+# Versioning scheme
+
+The SpatialOS GDK for Unreal consists of:
+
+*   [The GDK Plugin](https://github.com/spatialos/unrealgdk)
+*   [The SpatialOS Unreal Engine Fork](https://github.com/improbableio/UnrealEngine)
+*   [The Example Project](https://github.com/spatialos/UnrealGDKExampleProject)
+
+Each of these follows the same versioning scheme:
```
[the remark this answers is not in the document]
Comment (ernopp) [reply SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md]: are you sure of this? https://www.grammar.com/each-singular-or-plural/

File: SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+For example: 
+
+*   [Version 0.5.0-preview](https://github.com/spatialos/UnrealGDK/releases/tag/0.5.0-preview) released on June 25th.
+*   [Version 0.4.2](https://github.com/spatialos/UnrealGDK/releases/tag/0.4.2) released on May 20th.
+
+## Which branches to use
+
+When setting up the GDK for your project, you need to decide which branch of each of the repositories (listed above) to clone.
+
+**The most stable version is always the `release` branch, which is set as the default branch for each repository.** We test this branch end-to-end with the SpatialOS SDKs, Tools, and Metagame Services, and it is fully documented. We update it a few times a year as part SpatialOS for Unreal milestones such as 2019.1.
```
[the remark this answers is not in the document]
Comment (ernopp) [reply SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md]: yep forwarded you an email on this

File: SpatialGDK/Documentation/content/upgrading.md (hunk 5 above)
Comment (ernopp) [reply SpatialGDK/Documentation/content/upgrading.md]: yes fixing

File: SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md (hunk 2 above)
Comment (ernopp) [reply SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md]: nice thanks

File: SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md (hunk 3 above)
Comment (ernopp) [reply SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md]: I agree that's better language, updating to that. 
It still is non-commital - I disagree that a non-commital statement is not helpful

File: SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+
+An alternative option is to use the `preview` branch. This branch contains the latest `*-preview` version, which we test using Improbable partner projects, but not as extensively as versions in `release`. Preview releases are for teams already developing on the GDK and working closely with Improbable.
```
[the remark this answers is not in the document]
Comment (ernopp) [reply SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md]: udpatre

File: SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (oblm) [reply SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md]: You're right, english is weird.

[the remark this answers is not in the document]
Comment (NotRobSheridan) [reply SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md]:
```suggestion
These repositories all follow the same versioning scheme, as described below: 
```

File: SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+For example:
+
+*   ``4.22-SpatialOSUnrealGDK-release`` (default)
+*   ``4.22-SpatialOSUnrealGDK-preview``
+
+when Unreal Engine 4.22 is the version supported.
+
+## Unreal Engine version support
+
+**The GDK has full support for one version of Unreal Engine at a time** - the version that is shown in the Unreal Engine Fork repository’s default branch name, and is indicated in the Get Started documentation.
```
[the remark this answers is not in the document]
Comment (ernopp) [reply SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md]: thanks this is better

File: SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md (hunk 1 above)
Comment (ernopp) [reply SpatialGDK/Documentation/content/pricing-and-support/versioning-scheme.md]: thx

**Pull request merged**